src/mlpack/methods/bayesian_ridge/CMakeFiles/CMakeDirectoryInformation.cmake

```diff
@@ -0,0 +1,16 @@
+# CMAKE generated file: DO NOT EDIT!
```

Comment:
Can you remove the ``CMakeFiles`` folder from the PR, I wonder why this even ends up here, looks like the files are auto-generated but I guess the gitignore doesn't catch those files, maybe it makes sense to update the gitignore, will take a look.

Reply:
I hope I have done the necessary. A new compilation after deleting the concerned folder do not make it appears again.